be/src/olap/memtable_flush_executor.cpp

```diff
@@ -0,0 +1,153 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "olap/memtable_flush_executor.h"
+
+#include "olap/data_dir.h"
+#include "olap/delta_writer.h"
+#include "olap/memtable.h"
+#include "runtime/exec_env.h"
+
+namespace doris {
+
+OLAPStatus FlushHandler::submit(std::shared_ptr<MemTable> memtable) {
```

Comment:
do we need to limit the number of memtables submitted to executor?
I'm afraid that one big ingest will block other ingest.

Reply:
Too many memtables in flush queue will take too much memory.
And the size of a memtable is no more than 100MB, so there will no be a `big ingest`